### ISSUE TITLE
Proposal:  Nominate Your Best Devops Tools

### ISSUE THREAD

**Issue by robmoffat** [finos/devops-automation#45]
@mcleo-d asked me to raise the proposal I brought up at the last meeting as a GitHub issue so we could discuss further.

## TLDR

Basically the proposal is:  can we pull together a list of (ideally OSS) tools that we recommend for DevOps?  I suggest we start off by writing our own top 5 (or so) in the comments below, and then maybe we can hash out what is common.  

## More Detailed Reasoning

The idea of DevOps Mutualization (it seems to me) is to gain value from building DevOps tooling together.  Therefore, it seems sensible to share (as far as possible) what tools we are already using.  That is - not _just_ trying to build new tools/extensions together, but trying to provide leadership / advice over what existing tools should be used in the first place.  

I did a brief search on GItHub to see if anyone else had "colonised" this space, and it seems like not.   Take a look at the DevOps topic on GitHub:

https://github.com/topics/devops

The top project on there with 62k stars is _somewhat_ related to DevOps... but not too much.  It's this one:  https://github.com/trimstray/the-book-of-secret-knowledge

I think we could make DevOps Mutualization quite a popular repo if it covered:
 - What tools to use to start your DevOps journey
 - As much detail as possible (within reason) about how we are using the tools
 - A place where we can propose new tools, or seed new DevOps tooling projects.

Anyway, I'd love to know what you all think about this.  I am new to all of this, and I find the best way to learn is to be wrong a lot.  So please tell me if that's the case!

**Comment by robmoffat** [comment]
A starter for 10  from me that have been top-of-mind lately:

1. GitHub Actions
2. Codecov.io
3. whitesourcesoftware.com
4. Spring Boot Micrometer https://www.baeldung.com/micrometer
5. https://grafana.com
6. Maven
7. TeamCity
8. Azure App Service
9. Twitter/Symphony/Teams Alerting 😄 
10. Git?  (sorry, I have a hard stop rn!)

**Comment by ashukla13** [comment]
@robmoffat  - thanks for raising this; i have added this to agenda for #46

**Comment by mcleo-d** [comment]
This was discussed on #46 to discuss async and to be taken into the next DevOps call.

**Comment by nikitakoselev** [comment]
@Jenkins - for their absolutely amazing step-by-step tutorial on contributions: https://www.jenkins.io/participate/
